src/VisualStudio/IntegrationTest/TestUtilities/Common/ProjectUtilities.cs

```diff
@@ -9,10 +9,13 @@ public abstract class Identity
 
     public class Project : Identity
     {
-        public Project(string name)
+        public Project(string name, string relativePath = null)
         {
             Name = name;
+            RelativePath = relativePath;
         }
+
+        public string RelativePath { get; }
```

Comment:
Also, if the `relativePath` argument passed in is `null`, should the `RelativePath` property just be set to the `Name`?